src/templates/weno/layout.html.twig

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %} {{ "Welcome"|xlt }} {% endblock %}</title>
+    {{ setupHeader(['jquery-ui', 'jquery-ui-sunny']) }}
+</head>
```

Comment:
since planning to remove jquery-ui https://github.com/openemr/openemr/issues/2999 prob shouldn't introduce more if there's another option